tests/integration/docker/user.bats

```diff
@@ -45,3 +45,22 @@ setup() {
 	[ "${status}" -eq 0 ]
 	[[ "${output}" == 'root'* ]]
 }
+
+@test "Run with additional groups" {
+	run $DOCKER_EXE run --rm --group-add audio --group-add nogroup busybox id
+	[ "${status}" -eq 0 ]
+	echo "${output}" | grep "audio"
+	echo "${output}" | grep "nogroup"
+}
+
+@test "Run with non-existing numeric gid" {
+	skip "This is not working"
+	# When passed a non-existent numeric id, hyperstart errors out saying that the
+	# user does not exist. This works with runc.
+	# Docker mentions "When passing a numeric ID, the user does not have
+	# to exist in the container."
+	# This needs to be fixed in hyperstart codebase.
```

Comment:
Was about to open one, but saw just now that there is an issue created for this already : https://github.com/hyperhq/hyperstart/issues/250.
It pretty much covers what we are seeing here. Will add the issue here as well.